packages/esl-website/views/playground/index.njk

```diff
@@ -0,0 +1,65 @@
+---
+  layout: content
+  title: Playground
+  order: 11
+  tags: [nav]
+
+  containerCls: uip-landing typo-large
+  landing: true
+  core-collection: playground
+  collection: playground
+
+  icon: sidebar/playground.svg
+  eleventyImport:
+    collections: ["playground"]
+---
+
+<div class="uip-landing-marquee container">
+  <div class="uip-landing-logo">
+    &lt;
+      <span class="uip-landing-logo-typewriter">UI Playground</span>
+    &gt;
+  </div>
+  <p class="uip-landing-description">{{playground.marquee.description}}</p>
+</div>
+
+<div class="uip-landing-benefits container">
+  <h2 class="uip-landing-heading">{{playground.benefits.title}}</h2>
+  <esl-animate target="::next::child" group></esl-animate>
+  <ul class="uip-landing-benefits-list">
+    {% for item in playground.benefits.items %}
+      <li class="uip-landing-benefits-list-item esl-animate-slide left">
+        {{ item.text }}
+      </li>
+    {% endfor %}
+  </ul>
+</div>
+
+<div class="uip-landing-d container">
+  <h2 class="uip-landing-heading">{{playground.demo.title}}</h2>
+  <p> {{playground.demo.description}} </p>
+
+  <uip-root store-key='uip-landing' editor-collapsed>
+    <script type="text/html" uip-snippet label="Logo">
+      <div class="uip-landing-d-logo">
+        <a class="uip-landing-d-logo-link blue" href="{{ '/playground/introduction/' | url }}">
+          <span>Get started with</span>
+            {% set snippetContent %}
+              {% include 'static/assets/sidebar/playground.svg' %}
+            {% endset %}
+            {{ snippetContent | replace('<svg', '<svg width="100"') | safe }}
```

Comment:
Mb we can achieve it differently, using a style tag or separtae/updated svg, instead of not so obvious replacing inline